openhgnn/experiment.py

```diff
@@ -78,7 +79,7 @@ def __init__(self, model, dataset, task,
         if use_best_config:
             self.config = set_best_config(self.config)
         self.set_params(**kwargs)
-        print(self)
```

Comment:
Do we need to delete this line? @Zhanghyi

Reply:
Dont delete it. This line prints experiment information.